examples/eks_cluster/variables.tf

```diff
@@ -21,11 +21,39 @@ variable "aws_secret_access_key" {
 variable "cluster_region" {
   type        = string
   description = "AWS region your cluster is located."
-
 }
 
 variable "cluster_name" {
   type        = string
   description = "EKS cluster name in AWS account."
 }
 
+# Addresses with whitelisted access to Kubernetes API
+variable "whitelisted_ips" {
```

Comment:
do we really need this?

Reply:
good point, we used to have this when we had public/private examples. will remove